.github/actions/build-backend/action.yml

```diff
@@ -180,10 +176,8 @@ runs:
         max_attempts: 2
         retry_wait_seconds: 30
         command: |
-          cd prime-router
-          export $(xargs < .vault/env/.env.local)
-          ./prime create-credential --type=UserPass --persist=DEFAULT-SFTP --user foo --pass pass
-          ./gradlew testSmoke
+          ./gradlew :prime-router:reloadCredentials
```

Comment:
I'm not sure why just this step wasn't using gradlew commands so I switched it over to make it simpler

Reply:
nice find